types/jsdom/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for jsdom 11.12
+// Type definitions for jsdom 12.2
```

Comment:
RresourceLoader was added in 12.0, the `element` option (see second linked PR) in 12.2